.github/workflows/deploy.yml

```diff
@@ -4,6 +4,9 @@ on:
   push:
     branches:
       - main
+  pull_request:
+    branches:
+      - main
```

Comment:
Am I understanding correctly that this would deploy everytime a PR is opened against main? would that be desired by default? 🤔